app/src/pages/inside/filtersPage/filtersPage.jsx

```diff
@@ -143,7 +149,7 @@ export class FiltersPage extends Component {
       ...rest
     } = this.props;
     return (
-      <PageLayout title={intl.formatMessage(messages.filtersPageTitle)}>
+      <PageLayout Title={this.getTitle()}>
```

Comment:
<PageLayout breadcrumbs={this.getBreadcrumbs()}>. Please rewrite all following cases in the same style